modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/SilverContextMenu.ts

```diff
@@ -119,22 +142,37 @@ export const isTriggeredByKeyboard = (editor: Editor, e: PointerEvent) =>
 const getSelectedElement = (editor: Editor, e: PointerEvent) =>
   isTriggeredByKeyboard(editor, e) ? editor.selection.getStart(true) : e.target as Element;
 
-const shouldUseNodeAnchor = (editor: Editor, e: PointerEvent) => {
+const getPointAnchorSpec = (editor: Editor, e: EditorEvent<TouchEvent>) => {
+  const pointAnchor = getPointAnchor(editor, e);
+  if (isTouch()) {
+    return {
+      bubble: Bubble.nu(0, bubbleSize, bubbleAlignments),
+      layouts,
+      overrides: {
+        maxWidthFunction: MaxWidth.expandable(),
+        maxHeightFunction: MaxHeight.expandable()
+      },
+      ...pointAnchor
+    };
+  } else {
+    return pointAnchor;
+  }
+};
+
+const getAnchorSpec = (editor: Editor, e): AnchorSpec => {
   const selector = Settings.getAvoidOverlapSelector(editor);
-  if (isTriggeredByKeyboard(editor, e)) {
-    return true;
-  } else if (selector) {
+  if (Type.isString(selector) && selector !== '') {
```

Comment:
This is a behavioural change. Are we sure we want that? If so it'll need to be included in the changelog.

Reply:
We also should be using `Strings.isNotEmpty(selector)` here instead of `selector !== ''`.

Reply:
Did that to avoid the truthy condition.